src/TLuaInterpreter.cpp

```diff
@@ -820,29 +815,27 @@ int TLuaInterpreter::wrapLine(lua_State* L)
 {
     int s = 1;
     int n = lua_gettop(L);
-    string a1 = "main";
+    string window = "main";
     if (n > 1) {
         if (!lua_isstring(L, s)) {
-            lua_pushstring(L, "wrapLine: wrong argument type");
-            lua_error(L);
-            return 1;
+            lua_pushfstring(L, "wrapLine: bad argument #1 type (MiniConsole name as string expected, got %s!)", luaL_typename(L, 1));
+            return lua_error(L);
         } else {
-            a1 = lua_tostring(L, s);
+            window = lua_tostring(L, s);
             s++;
         }
     }
 
-    int luaNumOfMatch;
+    int lineNumber;
     if (!lua_isnumber(L, s)) {
-        lua_pushstring(L, "wrapLine: wrong argument type");
-        lua_error(L);
-        return 1;
+        lua_pushfstring(L, "wrapLine: bad argument #2 type (line as number expected, got %s!)", luaL_typename(L, s));
+        return lua_error(L);
     } else {
-        luaNumOfMatch = lua_tointeger(L, s);
+        lineNumber = lua_tointeger(L, s);
     }
 
     Host& host = getHostFromLua(L);
-    host.mpConsole->luaWrapLine(a1, luaNumOfMatch);
+    host.mpConsole->luaWrapLine(window, lineNumber);
```

Comment:
Does `TConsole::luaWrapLine(`...`)` have a return value (`bool` perhaps) to report back - say because it could or couldn't find the window name given.

Reply:
It's `void`